Tests.Integration.Presentation.Web/Advice/AdviceTest.cs

```diff
@@ -110,5 +221,10 @@ private string createWellformedEmail()
         {
             return $"{A<string>()}@test.dk";
         }
+
+        private DateTime getRandomDateAfterToday()
+        {
+            return DateTime.Now.AddDays(A<int>());
```

Comment:
@mbkstrongminds Bær OBS på at A<int> returnerer også negative tal. For at gøre din test stabil bør du måske ændre den til `Math.Abs(A<int>)`